wso2is_analytics/hieradata/dev/wso2/wso2is_analytics/pattern-1/default.yaml

```diff
@@ -0,0 +1,417 @@
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+classes:
+  - wso2is_analytics
+
+wso2::service_name: wso2is_analytics
+wso2::hostname: is.analytics.dev.wso2.org
+wso2::mgt_hostname: is.analytics.dev.wso2.org
+wso2::pack_filename: wso2is-analytics-5.4.0.zip
+
+#wso2::file_list:
+#  - "repository/components/lib/%{hiera('wso2::datasources::mysql::connector_jar')}"
+#wso2::datasources::mysql::connector_jar: mysql-connector-java-5.1.39-bin.jar
+
+wso2::template_list:
+  - repository/conf/identity/identity.xml
+  - repository/conf/datasources/analytics-datasources.xml
+  - repository/conf/datasources/metrics-datasources.xml
+  - repository/conf/analytics/spark/spark-defaults.conf
+  - repository/conf/event-processor.xml
+  - repository/deployment/server/jaggeryapps/portal/configs/designer.json
+#  - repository/conf/security/cipher-text.properties
+#  - repository/conf/security/cipher-tool.properties
+#  - bin/ciphertool.sh
+#  - password-tmp
```

Comment:
Secure Vault related files are not templated anymore as per the latest `wso2base` module.

Reply:
Since the latest released version is 1.0.1 we will be depending on it